### PR TITLE
NMS-10671: Default load threshold contains calculation error

### PR DESCRIPTION
**Pull request by mfuhrmann**
Since load metric has to be divided by 100.

* JIRA: https://issues.opennms.org/browse/NMS-10671

### CODE REVIEW

**Comment by christianpape** [comment]
Since this is a bug fixes for release-24.0.0 it must go into release-24.1.0.

**Comment by christianpape** [comment]
Fixed the branch.

File: opennms-base-assembly/src/main/filtered/etc/thresholds.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
...
       <threshold description="Trigger an alert when the percentage of inodes used on any disk increases by a relative 33.3% compared to its most recent previous measurement (that is, there are suddenly fewer free inodes)" type="relativeChange" ds-type="dskIndex" value="1.333" rearm="0.0" trigger="1" ds-label="ns-dskPath" filterOperator="OR" ds-name="ns-dskPercentNode"/>
-      <expression description="Trigger an alert when the five minute CPU load average metric reaches or goes above 70% for two consecutive measurement intervals" type="high" ds-type="node" value="70.0" rearm="50.0" trigger="2" filterOperator="OR" expression="(loadavg5 / CpuNumCpus ) * 100.0"/>
+      <expression description="Trigger an alert when the five minute CPU load average metric reaches or goes above 70% for two consecutive measurement intervals" type="high" ds-type="node" value="70.0" rearm="50.0" trigger="2" filterOperator="OR" expression="(loadavg5 / 100) / CpuNumCpus )) * 100.0"/>
```
Comment (indigo423) [inline opennms-base-assembly/src/main/filtered/etc/thresholds.xml]: the number of brackets in the expression looks strange to me :) 1 open and 3 closings :)

`(loadavg5 / 100) / CpuNumCpus )) * 100.0`

**Comment by christianpape** [comment]
You're right. I just copied the line into my OpenNMS instance and it worked. But yes, the brackets are wrong.

**Comment by christianpape** [comment]
Brackets fixed.

**Comment by mfuhrmann** [comment]
Thanks @christianpape @indigo423

**Comment by mvrueden** [comment]
I am going ahead and merge this.
Feel free to do this by yourself next time if a consent has been reached and all issues have been addressed.

**Pull request merged**